Rakefile

```diff
@@ -7,4 +7,5 @@ Osra::Application.load_tasks
 
 require 'coveralls/rake/task'
 Coveralls::RakeTask.new
-task :test_with_coveralls => [:spec, :cucumber, 'coveralls:push']
+#task :test_with_coveralls => [:cucumber_hq, 'coveralls:push']
+task :test_with_coveralls => [:spec, :cucumber_hq, :cucumber, 'coveralls:push']
```

Comment:
I get `[Coveralls] Outside the Travis environment, not sending data.` after running `$ rake test_with_coveralls`. 
This happens after adding  the `:cucumber_hq`  tests to this `:test_with_coveralls` task. 
Not sure what else I have to configure for coveralls.